packages/tools/fuzzer/ipc/Server.ts

```
@@ -0,0 +1,145 @@
+			stdin: 'ignore', // No stdin
+			// For debugging:
+			stderr: process.stderr,
+			// stdout: process.stdout,
+		});
+	}
+
+	private static assertCanUseSnapshots_() {
+		if (process.env.SQLITE_DATABASE) {
+			throw new Error(`Unsupported: Creating snapshots of a non-default database (${JSON.stringify(process.env.SQLITE_DATABASE)}) is not supported. Skipping...`);
+		}
+		if (process.env.DB_CLIENT ?? 'sqlite' !== 'sqlite') {
```

Biome (2.3.14)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)